plugins/woo-ai/api/api.php

```diff
@@ -17,7 +17,8 @@
  * Register the Woo AI route.
  */
 function woo_ai_rest_api_init() {
+	error_log( 'this one is not' );
```

Comment:
As noted here, for some reason none of my `error_log`'s are showing up for the API logic.